exercises/03_reverse/reverse.go

```diff
@@ -1,4 +1,12 @@
 package main
 
+func reverse(n int) int {
+	result := 0
+	for ; n != 0; n /= 10 {
+		result = (result + (n % 10)) * 10
+	}
+	return result / 10
```

Comment:
Please try to update your program in a way to get rid of this division. For that you have to change line 6.